runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -176,30 +177,53 @@ def model(self, id: str) -> Optional[prophet.forecaster.Prophet]:
             return model_from_json(self._model_json[id])
         return None
 
-    def _make_future_dataframe(self, id: str, horizon: int, include_history: bool = True) -> pd.DataFrame:
+    def make_future_dataframe(
```

Comment:
I'm a bit confused why we need both this method and the util.make_future_dataframe. Is this method used anywhere?

Reply:
Adding the function to the model because the model has the infos like start and end time, horizon, frequencies. The actual implementation is in util so that the two models can use the same one.

Reply:
It is designed to be used in the trial notebook for model inference with extra features. We will generate the future dataframe, join feature store tables and predict the future data.

Reply:
ahhh got it. That makes sense! Thanks for providing the context!